lib/wallet/integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1729,12 +1729,13 @@ fundSharedWallet ctx amt sharedWals = do
    let ep = Link.createTransactionOld @'Shelley
    rTx <- request @(ApiTransaction n) ctx (ep wShelley) Default payloadTx
    expectResponseCode HTTP.status202 rTx
+   let fee = unsafeResponse rTx ^. #fee . #getQuantity
    eventually "wShelley balance is decreased" $ do
        ra <- request @ApiWallet ctx
            (Link.getWallet @'Shelley wShelley) Default Empty
        expectField
            (#balance . #available)
-           (`shouldBe` Quantity (faucetAmt - feeMax - amt)) ra
+           (`shouldBe` Quantity (faucetAmt - fee - amt)) ra
```

Comment:
Perhaps this change was accidentally included in this PR?

On my machine, compilation of `Framework.DSL` fails with:
```
integration/src/Test/Integration/Framework/DSL.hs:1727:38: error: [-Wunused-matches, -Werror=unused-matches]
    Defined but not used: `feeMax'
     |
1727 |    (_, ApiFee (Quantity _) (Quantity feeMax) _ _) <- unsafeRequest ctx
```